app/controllers/admin/deals_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::DealsController < Admin::BaseController
+  before_action :requested_deal, only: [:show, :edit, :destroy, :update]
+  
+  def index
+    @deals = Deal.where(status: params[:status])
+    @status = params[:status]
+  end
+
+  def new
+    @deal = Deal.new
+  end
+
+  def create
+    deal = Deal.new(deal_permitted_params)
+    deal.admin = logged_in_user
+
+    if deal.save
+      redirect_to admin_deals_url(status: 'unpublished')
+    else
+      redirect_to new_admin_deal_url, notice: t(:default_error_message)
+    end
+  end
+
+  def show
+    render template: 'admin/deals/show'
```

Comment:
what happens if we remove this line?

Reply:
It will work fine, here it doesn't matter whether we write this or not.
It will by default render show template if we don't write it.
Removed this line